Pixel Trees

diicasses opened the pull request:
Added Pixel Trees Policy ID
https://www.anti.biz/pixeltrees

MaxJewell commented:
There is a small mistake with your policy ID.
There is an additional '8' on the end of the policy ID on your website. 
When this is sorted out, the request can go through.

Thanks.

diicasses commented:
Okay I will fix it 

Sent from my iPhone

> On Sep 21, 2021, at 06:16, MaxJewell ***@***.***> wrote:
> 
> ﻿
> There is a small mistake with your policy ID.
> There is an additional '8' on the end of the policy ID on your website.
> When this is sorted out, the request can go through.
> 
> Thanks.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android.

diicasses commented:
> There is a small mistake with your policy ID.
> There is an additional '8' on the end of the policy ID on your website.
> When this is sorted out, the request can go through.
> 
> Thanks.

![image](https://user-images.githubusercontent.com/79003733/134160216-d00a9d2a-3345-4fcc-8971-fe7c5c02e213.png)
I fixed it, also I have a new project https://www.ooz1es.io I am going to submit the merge for that one as well.

diicasses commented:
https://github.com/diicasses/policyIDs/blob/main/ooz1es I changed it to main

On Tue, Sep 21, 2021 at 7:12 AM M W ***@***.***> wrote:

> I fixed the PIXEL TREES mistake on my website anti.biz/pixeltrees
>
> I also created a merge request for
> https://github.com/diicasses/policyIDs/blob/patch-1/ooz1es#L1
>
> My new project at www.ooz1es.io
>
> On Tue, Sep 21, 2021 at 6:16 AM MaxJewell ***@***.***>
> wrote:
>
>> There is a small mistake with your policy ID.
>> There is an additional '8' on the end of the policy ID on your website.
>> When this is sorted out, the request can go through.
>>
>> Thanks.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/Cardano-NFTs/policyIDs/pull/626#issuecomment-923837961>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AS2YAVMI3ZMSBNBXZ7VGM63UDBLQTANCNFSM5DE62LDQ>
>> .
>> Triage notifications on the go with GitHub Mobile for iOS
>> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
>> or Android
>> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>>
>>
>
>
> --
> www.anti.biz
>
>

-- 
www.anti.biz

MaxJewell commented:
I  will need links to the new policy ID's you have put through.
Thanks

diicasses commented:
Thanks for merging Pixel Trees 

Any chance to merge OOZ1ES & Space Babez ?

Sent from my iPhone

> On Oct 5, 2021, at 11:53, MaxJewell ***@***.***> wrote:
> 
> ﻿
> Merged #626 into main.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android.
